.github/workflows/tests.yaml

```diff
@@ -5,56 +5,28 @@ on:
       - '**'
 env:
    COPYRIGHT_COMPANY: 'NetCracker Technology Corporation'
-   COPYRIGHT_YEAR: '2021-2022'
+   COPYRIGHT_YEAR: '2021-2023'
 jobs:
   license:
      runs-on: ubuntu-latest
      steps:
-       - uses: actions/checkout@v2
+       - uses: actions/checkout@v3
        - run: docker run -v "${PWD}:/src" -i ghcr.io/google/addlicense -v -c "${{ env.COPYRIGHT_COMPANY }}" -y "${{ env.COPYRIGHT_YEAR }}" $(find . -type f -name "*.py" | xargs echo)
        # This currently not works for protected branches,
        # but all pushes to them should be done through PRs that already contain inserted license.
        - uses: stefanzweifel/git-auto-commit-action@v4
          with:
            commit_message: Auto-update license header
-  selftest-image:
-    needs: license
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout Repo
-        uses: actions/checkout@v2
-      - name: Build Docker Image
-        run: docker build -t kubemarine --no-cache .
-      - name: Run Selftest
-        # selftest of image internally runs selftest of package
-        run: docker run kubemarine selftest
-  selftest-binary:
-    needs: license
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout Repo
-        uses: actions/checkout@v2
-      - name: Build Binary via Docker
-        run: |
-          docker build -t kubemarine --build-arg BUILD_TYPE=binary --no-cache .
-          CONTAINER_ID=$(docker create kubemarine)
-          docker cp "${CONTAINER_ID}:/opt/kubemarine/dist/kubemarine" kubemarine_exe
-          docker rm -v "${CONTAINER_ID}"
-      - name: Run Selftest
-        run: ./kubemarine_exe selftest
-
   unit-tests:
-    needs: license
```

Comment:
It was actually a useless dependency as all jobs in workflow are anyway run on the same commit.